.codecov.yml

```diff
@@ -3,10 +3,5 @@ coverage:
   round: up
   range: "0...100"
   status:
-    project:
-      default:
-        if_ci_failed: success
-    patch:
-      default:
-        if_ci_failed: success
-
+    project: false
```

Comment:
@karelhala I don't think we want to undo this change in this PR?

Reply:
@florkbr the runs were still breaking the CI. This disables the runs altogether, will post the comment about status, but won't run the jobs. I got fed up with the config and added it here. I could create a new PR if you want to.